lib/pengine/complex.c

```diff
@@ -683,13 +683,24 @@ common_unpack(xmlNode * xml_obj, resource_t ** rsc,
     }
 
     pe_rsc_trace((*rsc), "\tRequired to start: %s%s", value, isdefault?" (default)":"");
-
     value = g_hash_table_lookup((*rsc)->meta, XML_RSC_ATTR_FAIL_TIMEOUT);
     if (value != NULL) {
         /* call crm_get_msec() and convert back to seconds */
         (*rsc)->failure_timeout = (crm_get_msec(value) / 1000);
     }
 
+    if (baremetal_remote_node) {
```

Comment:
Do we need to guard it?
Could we not just unpack it unconditionally and leave it unused if its not a remote node?